native/src/routes/Events.tsx

```diff
@@ -57,8 +60,6 @@ const Events = ({ cityModel, language, navigateTo, events, slug, refresh }: Even
   }
 
   if (slug) {
-    const event = events.find(it => it.slug === slug)
```

Comment:
why this change was needed?

Reply:
for useTtsPlayer and for less redundant code..
![image](https://github.com/user-attachments/assets/0579d765-0c6b-4d6e-96d7-38ee9b0e883d)